package.json

```diff
@@ -79,7 +79,7 @@
     "webpack": "^5.59.1"
   },
   "engines": {
-    "node": "12.* || >= 14.*"
+    "node": "14.* || >= 16.*"
```

Comment:
@chriskrycho should we actually do `14.* || 16.* || >= 18.*`?

Reply:
Ah, yep.

Reply:
Done!